charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -53,12 +53,17 @@ spec:
         command:
         - /var/etcd/bin/bootstrap.sh
         readinessProbe:
-          httpGet:
+          exec:
+            command:
+            - /usr/bin/curl
 {{- if .Values.etcd.enableTLS }}
-            scheme: HTTPS
-{{- end }}
-            path: /healthz
```

Comment:
Please use the `/healthz` for single node etcd clusters because this endpoint also considers owner checks.

Reply:
Done